src/main/java/com/github/piedpiper/node/stepfunctions/StepFunctionsServiceNode.java

```diff
@@ -0,0 +1,45 @@
+package com.github.piedpiper.node.stepfunctions;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+import com.github.piedpiper.common.PiedPiperConstants;
+import com.github.piedpiper.node.BaseNode;
+import com.github.piedpiper.node.NodeInput;
+import com.github.piedpiper.node.NodeOutput;
+import com.github.piedpiper.node.ParameterMetadata;
+import com.google.common.collect.Lists;
+
+public class StepFunctionsServiceNode extends BaseNode {
+
+	public static final ParameterMetadata METHOD = new ParameterMetadata("method", ParameterMetadata.MANDATORY,
+			Lists.newArrayList("EXECUTE"));
+
+	@Override
+	public NodeOutput apply(NodeInput nodeInput) {
+		return getStepFunctionsMethodHandler(nodeInput).apply(nodeInput);
+	}
+
+	private Function<NodeInput, NodeOutput> getStepFunctionsMethodHandler(NodeInput nodeInput) {
+		try {
+			String method = getStepFunctionsMethod(nodeInput);
+			switch (method) {
+			case "EXECUTE":
+				return injector.getInstance(StepFunctionsExecute.class);
+
+			default:
+				throw new IllegalArgumentException(String.format("Unsupported StepFunctions Method = %s", method));
+			}
+		} catch (Exception e) {
+			throw e;
```

Comment:
Bad idea. Don;t catch and throw exception. has no meaning. When uyou catch you have to handle it. else catch and throw is the same as not having catchin and throw